looprpc/client.yaml

```diff
@@ -26,8 +26,8 @@ http:
       get: "/v1/loop/in/probe/{amt}"
     - selector: looprpc.SwapClient.GetInfo
       get: "/v1/loop/info"
-    - selector: looprpc.SwapClient.GetLsatTokens
-      get: "/v1/lsat/tokens"
+    - selector: looprpc.SwapClient.GetL402Tokens
+      get: "/v1/l402/tokens"
```

Comment:
Should we keep the old mapping (can have `additional_bindings` to map multiple URIs to the same method, see https://github.com/lightninglabs/taproot-assets/blob/main/taprpc/taprootassets.yaml#L66 for example) to not break any client applications?

Reply:
I added `additional_bindings`.

But it does not provide backward compatibility for GRPC. When I run new loopd with old loop, I get the following error, trying to to run `loop listauth`:
```
[loop] rpc error: code = Unimplemented desc = unknown method GetLsatTokens for service looprpc.SwapClient
```

Reply:
Yeah, this might be a problem for some web UIs that use the RPC method directly, like Terminal Web (see my comment above).